Makefile

```diff
@@ -80,61 +83,47 @@ GOTEST_INTEGRATION = $(GOTEST) -tags=integration
 
 # Integration test for sdk client
 .PHONY: test-sdk
-test-sdk: clean-sdk build-sdk
+test-sdk: clean-all build-all
```

Comment:
should this (and `-short`) be part of Makefile.sdk?

Reply:
I was thinking about it.. but may be this can be done while migrating to SDK?

I.e I still feel that there is still value in running all tests from short `make test` rather then not forgetting to add `-f ...`. It can be solved by include, etc,  but it's a bit more work than I would like to include here.

So decided to postpone, WDYT?

Reply:
:+1: we can rethink this later